reseq_ros2/description/macros/wheel.xacro

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0"?>
+<robot xmlns:xacro="http://www.ros.org/wiki/xacro">
+    <!--   
+        Creates one wheel of a chassis given its position
+
+        @param id: the id of the chassis (starting from 1)
+        @param pos: front/back
+        @param side: left/right
+    -->
+
+    <xacro:macro name="wheel_factory" params="id pos side">
+
+        <xacro:property name="pos_sign" value="1" />
+        <xacro:if value="${pos == 'back'}">
+            <xacro:property name="pos_sign" value="-1" />
+        </xacro:if>
+
+        <xacro:property name="side_sign" value="1" />
+        <xacro:if value="${side == 'right'}">
+            <xacro:property name="side_sign" value="-1" />
+        </xacro:if>
+
+        <joint name="${side}_${pos}_wheel_${id}_joint" type="continuous">
+        <parent link="chassis${id}_link" />
+        <child link="${side}_${pos}_wheel_${id}_link" />
+        <origin xyz="${pos_sign*WheelBase/2} ${side_sign*(ModuleWidth/2-TrackWidth/2)} 0" rpy="0 0 0" />
+        <axis xyz="0 0 ${side_sign}" />
```

Comment:
```suggestion
        <axis xyz="0 1 0" />
```
Since we aligned the axes, we have to change the axis around which the wheel rotates about. Before it was the Z axis (depending on the side). Now it is always the Y axis